Objective-C/Tests/URLEndpointListenerTest.m

```diff
@@ -624,16 +624,20 @@ - (void) testServerCertVerificationModeCACert {
     [self stopListener: listener];
 }
 
-- (void) _testEmptyNetworkInterface {
+- (void) testEmptyNetworkInterface {
     if (!self.keyChainAccessAllowed) return;
     
     [self listen];
-    
     NSArray* urls = _listener.urls;
+    
+    /** Link local addresses cannot be assigned via network interface because they don't map to any given interface.  */
+    NSPredicate* p = [NSPredicate predicateWithFormat: @"NOT(SELF.host BEGINSWITH 'fe80::')"];
```

Comment:
Wondering if LiteCore should filter this out?

Reply:
https://github.com/couchbase/couchbase-lite-core/blob/master/REST/tests/SyncListenerTest.cc#L184